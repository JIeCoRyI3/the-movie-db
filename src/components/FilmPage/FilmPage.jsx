import React from 'react';
import Header from '../Header/';
import MainContainer from '../MainContainer';
import PropTypes from 'prop-types';
import CurrentFilmDescription from '../CurrentFilm';

function FilmPage(props) {
    console.log(props.match.params);
    return (
        <div className="filmPage">
            <Header>
                <CurrentFilmDescription />
            </Header>
            <MainContainer />
        </div>
    );
}

// function FilmInfo() {
//     return <div>Film`&apos;`s info component!</div>;
// }

FilmPage.propTypes = {
    match: PropTypes.object,
};

export { FilmPage };
