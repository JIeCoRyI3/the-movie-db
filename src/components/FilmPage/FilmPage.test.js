import React from 'react';
import { shallow } from 'enzyme';
import { FilmPage } from './FilmPage';

describe('<FilmPage>', () => {
    it('should render <FilmPage/> correctly', () => {
        const component = shallow(<FilmPage />);
        expect(component).toMatchSnapshot();
    });
});
