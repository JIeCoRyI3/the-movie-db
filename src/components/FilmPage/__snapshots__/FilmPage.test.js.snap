// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmPage> should render <FilmPage/> correctly 1`] = `
<div
  className="filmPage"
>
  <withRouter(Connect(Header))>
    <withRouter(CurrentFilmDescription) />
  </withRouter(Connect(Header))>
  <withRouter(Connect(MainContainer)) />
</div>
`;
